ansible/condor.yml

```diff
@@ -45,58 +45,74 @@
         url: https://get.htcondor.org
         dest: /root/condor_install.sh
         mode: "0755"
-
-    - ansible.builtin.user:
-        user: clauncher
-        create_home: yes
-        append: true
-        groups: docker
   tags:
     - worker
     - manager
     - submission
 
-- name: install the central manager
+- name: install condor
   become: true
-  hosts: condor_manager
+  hosts: condor
   tasks:
+    - ansible.builtin.set_fact:
+        install_command:
+          - "./condor_install.sh"
+          - "--no-dry-run"
+          - "--password"
+          - "{{ condor_pool_password }}"
+        cm_opts:
+          - "--central_manager"
+          - "{{ groups['condor_manager'][0] }}"
+        exec_opts:
+          - "--execute"
+          - "{{ groups['condor_manager'][0] }}"
+        submit_opts:
+          - "--submit"
+          - "{{ groups['condor_manager'][0] }}"
+
+    - ansible.builtin.set_fact:
+        install_command: "{{ install_command + cm_opts }}"
+      when: "'condor_submit' in group_names"
```

Comment:
This looks wrong -- shoudn't this say `when: "'condor_manager' in group_names"`? When `condor_submit` is in the group names it's added to a couple lines below this, so it'll still work for submit nodes, but it looks like this would make all the submit nodes into central managers too.

Reply:
Yes it should! Thanks for catching that!